packages/lib/constants.ts

```diff
@@ -36,3 +36,8 @@ export const DEVELOPER_DOCS = "https://developer.cal.com";
 export const SEO_IMG_DEFAULT = `${WEBSITE_URL}/og-image.png`;
 export const SEO_IMG_OGIMG = `${CAL_URL}/api/social/og/image`;
 export const SEO_IMG_OGIMG_VIDEO = `${WEBSITE_URL}/video-og-image.png`;
+export const IS_STRIPE_ENABLED = !!(
+  process.env.STRIPE_CLIENT_ID &&
+  process.env.NEXT_PUBLIC_STRIPE_PUBLIC_KEY &&
+  process.env.STRIPE_PRIVATE_KEY
+);
```

Comment:
@zomars replace the IS_HOSTED_CAL with this constant for payment specific conditions

Reply:
Self hosted enterprise customers should be able to create teams freely unless they decide to charge for it. They might have stripe enabled for paid bookings, we cannot assume they will use this for teams as well.

Reply:
> > We also need to use the stripeCustomerId from the OWNER when creating the checkout session. Otherwise we will create duplicate customers with the same email.
> 
> My thought is that team owners & admins should be able to manage the team's subscription. So even though we may have the same email, team names will be unique. Also, people come and go from organizations teams should be able to change subscription information if that person leaves.

And how do an admin/other owners manage to uniquely identify the team on Stripe?


> > Cancelling the checkout also creates the team regardless we need to prevent that.
> 
> Teams are created after the first step of the new team form. This is so invites are associated with that team. One of my next PRs is to disable teams and have a message appear when team subscriptions are `incomplete`, `canceled`, and `unpaid`.

This will allow for team name poaching. Let's avoid creating the team until the checkout succeeds.